source/frontend/src/features/leases/add/AddLeaseForm.tsx

```diff
@@ -58,6 +61,14 @@ const AddLeaseForm: React.FunctionComponent<React.PropsWithChildren<IAddLeaseFor
             <LeaseDetailSubForm formikProps={formikProps}></LeaseDetailSubForm>
             <LeasePropertySelector formikProps={formikProps} />
             <AdministrationSubForm formikProps={formikProps}></AdministrationSubForm>
+            <Section header="Lease Team">
+              <AddLeaseTeamSubForm />
```

Comment:
nit, the form component could have the section and the error handling internally.